products/devsuite/_common/product.yml

```diff
@@ -1,7 +1,7 @@
 name: Red Hat Development Suite
-abbreviated_name: DevSuite
+abbreviated_name: Red Hat Development Suite
 product_logo: enterprise-app-platform.png
-current_version: 7.0.0
+current_version: 1.1.0
```

Comment:
I changed that because they were discussing 1.1. I was wondering why this was listed as 7.0